### ISSUE TITLE
comet init --api-key

### ISSUE THREAD

**Issue by kuri54** [comet-ml/issue-tracking#439]
Hi, 
I tried to register an API Key from the terminal using `comet init --api-key`, but the following message was displayed and I could not proceed.

`COMET INFO: Comet API key is not set. You'll have to provide it explictly to experiments and API`

I was able to do this at least until a month ago, what could be the cause?

OS: macOS 11.6
comet version: 3.19.1

**Comment by DN6** [comment]
@kuri54  Could you share the stack trace with the error you receive while trying to set your API Key?

**Comment by kuri54** [comment]
Today, I tried again to `export COMET_API_KEY=<Your API Key>` from the command line.
Last time I got an error message, but this time it worked fine.

Also, when I typed `comet init --api-key` on the command line, I did not get the same error as last time.
Maybe I was in a bad environment. (But I didn't change anything, and the error kept coming up for a few days.

Thank you for your support.
It looks like I can record the experiment for now.
